src/test/java/keycontacts/model/lesson/CancelledLessonTest.java

```diff
@@ -0,0 +1,48 @@
+package keycontacts.model.lesson;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.junit.jupiter.api.Test;
+
+public class CancelledLessonTest {
+    private Date date = new Date("06-07-2002");
+    private CancelledLesson cancelledLesson = new CancelledLesson(date);
+
+    @Test
+    public void getLessonDateTest() {
+        assertEquals(date, cancelledLesson.getLessonDate());
+        assertNotEquals(new Date("15-12-2002"), cancelledLesson.getLessonDate());
```

Comment:
This second line feels like you are testing the `.equals(Object other)` method of `Date`, which should be done in its own test file